test/e2e/fixture/applicationsets/actions.go

```diff
@@ -127,6 +159,103 @@ func (a *Actions) Create(appSet v1alpha1.ApplicationSet) *Actions {
 	return a
 }
 
+// Create a ConfigMap for the ClusterResourceList generator
+func (a *Actions) CreateConfigMap(configMapName string) *Actions {
```

Comment:
This should be renamed to something like `CreatePlacementDecisionConfigMap`, so as not to be confused with a method that does generic ConfigMap creation. 

(Or you could turn it into a generic config map creator, taking a `*corev1.ConfigMap` as a parameter, but renaming it is probably easier)

Reply:
Went with your `CreatePlacementDecisionConfigMap`